packages/dcos-integration-test/extra/test_metrics.py

```diff
@@ -100,22 +100,53 @@ def test_metrics_containers(dcos_api_session):
     def test_containers(app_endpoints):
         for agent in app_endpoints:
             response = dcos_api_session.metrics.get('containers', node=agent.host)
+            assert response.status_code == 200, "expected 200 status code from /containers/, got {}".format(
```

Comment:
It's pretty standard throughout the integration tests to drop the message bit as it is very clear from the context that the request to `/containers` failed and the assert will print the actual `response.status_code`